src/boosting/score_updater.hpp

```diff
@@ -85,7 +85,7 @@ class ScoreUpdater {
   * \param tree_learner
   * \param cur_tree_id Current tree for multiclass training
   */
-  inline void AddScore(const TreeLearner* tree_learner, const Tree* tree, int cur_tree_id) {
+  inline void AddScore(TreeLearner* tree_learner, const Tree* tree, int cur_tree_id) {
```

Comment:
tree learner will be changed in score updating?

Reply:
Sorry, I think this was from an older approach and I forgot to change it back. Fixed now.